lxd/db/storage_volumes.go

```diff
@@ -621,6 +627,8 @@ type StorageVolumeArgs struct {
 	ProjectName string
 
 	ContentType string
+
+	NodeID sql.NullInt64
```

Comment:
@monstermunchkin In NodeID is normally an int64, so I'd stick with that convention. What I've done in the past is to coalesce nulls to -1 as no valid entry will be <0